test/end-to-end/test/test_createRecipe.js

```diff
@@ -20,16 +22,28 @@ describe('Create Recipe Page', () => {
   const createRecipePage = new CreateRecipePage(pageConfig.recipe.simple.name
     , pageConfig.recipe.simple.description);
 
+// Switch to welder-web iframe if welder-web is integrated with Cockpit.
+// Cockpit web service is listening on TCP port 9090.
+  beforeEach(() => {
```

Comment:
I see that we have the same `beforeEach` block in several test files. Can you instead make it a helper function and only call the function in each test file?

Reply:
Added a helper, and call helper inside beforeEach.

Reply:
+1 on this